HS9_py2/heatsource9/Stream/StreamNode.pyx

```diff
@@ -472,51 +473,6 @@ def CalcDispersion(self):
         Disp = 0.1
         return Disp
 
-    def MacCormick_THW(self, time):

```

Comment:
I tend to remove unused code, rather than keep it. Less code increases maintainability. We have the old code in the Git history if we need it.